package.json

```diff
@@ -15,6 +15,7 @@
     "test:ci": "jest --clearCache && lerna run test:ci",
     "changeset": "changeset",
     "postinstall": "lerna bootstrap",
+    "version": "changeset version && npm i --package-lock-only",
```

Comment:
I think this will only update the lockfile for the root package-lock.json. We might need either a script or a lerna command to handle versioning each package's lock file (similar to how lerna lets you run tests for every child package that has a test command iirc). [Lerna 5 has these docs on it](https://lerna.js.org/docs/features/run-tasks).